Dockerfile

```diff
@@ -1,53 +1,50 @@
-FROM rocker/r-ver:4.3.1
+FROM rocker/r-ver:4.3.2
+
+# Set the working directory to setup. Uses a dedicated directory instead of
+# root since otherwise renv will try to scan every subdirectory
+WORKDIR /setup
 
 # Use PPM for binary installs
 ENV RENV_CONFIG_REPOS_OVERRIDE "https://packagemanager.posit.co/cran/__linux__/jammy/latest"
+ENV RENV_CONFIG_SANDBOX_ENABLED FALSE
 ENV RENV_PATHS_LIBRARY renv/library
+ENV RENV_PATHS_CACHE /setup/cache
```

Comment:
This is the actual location in the container the packages get installed to. They're then symlinked in the actual working dir.